README.md

```diff
@@ -30,3 +30,14 @@ Update `config/dev.webpack.config.js` according to your application URL. [Read m
 ### Testing
 
 `npm run verify` will run `npm run lint` (eslint) and `npm test` (Jest)
+
+### Run locally with chrome service BE
+
+!Prerequisites!
+Podman, podman-compose and go installed
```

Comment:
Maybe add a link to gvm. Some distros come with go installed but it might not be the right version.